modules/database/audit-autoloaded-options/hooks.php

```diff
@@ -20,9 +20,119 @@
  */
 function perflab_aao_add_autoloaded_options_test( $tests ) {
 	$tests['direct']['autoloaded_options'] = array(
-		'label' => __( 'Autoloaded options', 'performance-lab' ),
+		'label' => esc_html__( 'Autoloaded options', 'performance-lab' ),
 		'test'  => 'perflab_aao_autoloaded_options_test',
 	);
 	return $tests;
 }
 add_filter( 'site_status_tests', 'perflab_aao_add_autoloaded_options_test' );
+
+/**
+ * Register custom REST API route for updating autoload.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_register_rest_route() {
+	register_rest_route(
+		'perflab-aao/v1',
+		'/update-autoload',
+		array(
+			'methods'  => 'POST',
+			'callback' => 'perflab_aao_update_autoload_rest',
+		)
+	);
+}
+add_action( 'rest_api_init', 'perflab_aao_register_rest_route' );
+
+/**
+ * Callback for updating autoload via REST API.
+ *
+ * @since n.e.x.t
+ *
+ * @param WP_REST_Request $request Request object.
+ * @return WP_REST_Response
+ */
+function perflab_aao_update_autoload_rest( $request ) {
+	$option_name = $request->get_param( 'option_name' );
+	$autoload    = $request->get_param( 'autoload' );
+	$value       = $request->get_param( 'value' );
+
+	if ( empty( $option_name ) || empty( $autoload ) ) {
+		return new WP_REST_Response(
+			array(
+				'success' => false,
+				'message' => esc_html__( 'Invalid option name or autoload value.', 'performance-lab' ),
```

Comment:
This shouldn't be escaped in this context since it isn't where it is output. It should be escaped right when printed.

```suggestion
				'message' => __( 'Invalid option name or autoload value.', 'performance-lab' ),
```